src/templates/sermons.scss

```diff
@@ -0,0 +1,30 @@
+.template-sermons {
+  .template-sermons__pagination {
+    margin-left: 0;
+    margin-top: 2em;
+    text-align: center;
+    li {
+      display: inline-block;
+      margin-right: .2em;
+
+      a {
+        color: #747474;
```

Comment:
It would be good to start extracting these colors into a `global.colors.scss` or something like that. Then when we want to start tweaking/reconciling color variations we won't have as many places to change.